cmd/sbrmi/src/lib.rs

```diff
@@ -0,0 +1,686 @@
+//! 0x50  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok
+//! 0x60  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok
+//! 0x70  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok  ok
+//!
+//! === MCE on thread 0x25 (37), bank 0 ===
+//!
+//!     MCA_CTL(0)     0x0000000000ffffff
+//!     MCA_STATUS(0)  0xb420000001130839
+//!                    |
+//!                    +---> Valid (VAL)                         = true
+//!                          Overflow (OVER)                     = false
+//!                          Uncorrected error (UC)              = true
+//!                          Error condition enabled (EN)        = true
+//!                          Misc error register valid (MISCV)   = false
+//!                          Address register valid (ADDRV)      = true
+//!                          Processor context corrupt (PCC)     = false
+//!                          Task content corrupt (TCC)          = false
+//!                          Syndrome register valid (SYNDV)     = true
+//!                          Deferred error (Deferred)           = false
+//!                          Poisoned data consumed (Poison)     = false
+//!                          Extended Error Code                 = 0x113
+//!                          MCA Error Code                      = 0x839
+//!
+//!     MCA_ADDR(0)    0x000ffffe14700008
+//!     MCA_MISC(0)    0xd010000000000000
+//!     MCA_CONFIG(0)  0x00000007000001fd
+//!                    |
+//!                    +---> Interrupt Enable (IntEn)            = false
+//!                          Deferred error type                 = 0x0
+//!                          MCAX enable (McaxEn)                = true
+//!                          MCA FRU text supported              = false
+//!                          Address LSB in MCA_STATUS           = true
+//!                          Deferred error status supported     = true
+//!                          System fatal error event supported  = true
+//!                          Deferred interrupt type supported   = true
+//!                          Deferred error logging supported    = true
+//!                          MCAX capable                        = true
+//!
+//!     MCA_SYND(0)    0x000000005c000002
+//!     MCA_IPID(0)    0x001000b000000000
+//!                    |
+//!                    +---> MCA bank type                       = 0x10
+//!                          Instance ID (high)                  = 0x0
+//!                          Hardware ID                         = 0xb0
+//!                          Instance ID (low)                   = 0x0
+//!
+//!     MCA_DESTAT(0)  0x0000000000000000
+//!     MCA_DEADDR(0)  0x0000000000000000
+//! ```
+//!
+//! CPU identification information as provided by the `cpuid` instruction
+//! can be retrieved by using the `--cpuid` option and specifying a
+//! desired target thread; full MCA inbformation can similarly be retrieved
```

Comment:
Typo:
```suggestion
//! desired target thread; full MCA information can similarly be retrieved
```
(requires rebuilding the README as well)

Reply:
Done